treebeard/al_tree.py

```diff
@@ -172,7 +172,7 @@ def dump_bulk(cls, parent=None, keep_ids=True):
             parent = serializable_cls.objects.get(pk=parent.pk)
 
         # a list of nodes: not really a queryset, but it works
-        objs = serializable_cls.get_tree(parent)
+        objs = serializable_cls.get_tree(parent, qset)
```

Comment:
:information_source: Only AL's `dump_bulk` needs to also call `get_tree()`.

If there were a way to apply `qset` inside `bulk_dump()`, without also applying it to `get_tree()`, then we could have all tests enabled and passing.